### PR TITLE
Add blog for office hours

### PR DESCRIPTION
**Pull request by rolinawu**
Create a blog post for the office hours

### CODE REVIEW

**Comment by evykassirer** [comment]
check out https://wics-site-pr-270.herokuapp.com/ to see how it's showing up

with markdown, you need to add an extra new line to actually make it a new line

File: content/F2016/F16-Office-Hours.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Title: Office Hours for Fall 2016
+Date: 2016-10-06 19:00
+Category: Blog
+Tags: Office Hours
+Author: Rolina Wu
+Summary: This is the regular Office hours for Fall 2016
+
+Monday:
+10am - 1 pm
```
Comment (evykassirer) [inline content/F2016/F16-Office-Hours.md]: maybe make this a list with bullet points instead?

**Comment by evykassirer** [comment]
I also wonder if it would be nice to add a little blurb about why people might want to visit the office, or I guess you could link to https://wics-site-pr-267.herokuapp.com/contact-us/ (there's a blurb on that page, but you have to scroll a bit)

File: content/F2016/F16-Office-Hours.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Title: Office Hours for Fall 2016
+Date: 2016-10-06 19:00
+Category: Blog
+Tags: Office Hours
+Author: Rolina Wu
+Summary: This is the regular Office hours for Fall 2016
```
Comment (annalorimer) [inline content/F2016/F16-Office-Hours.md]: Make this something like "Announcing our Office Hours for Fall 2016"?

File: content/F2016/F16-Office-Hours.md [hunk 3]
```diff
@@ -3,7 +3,10 @@ Date: 2016-10-06 19:00
 Category: Blog
 Tags: Office Hours
 Author: Rolina Wu
-Summary: This is the regular Office hours for Fall 2016
+Summary: Announcing our Office Hours for Fall 2016
+
+Come visit us during our Office hours or just swing by to say hi! We have various technical interview resources and publications available for you to read during Office Hours. There's always an officer that will be available in the office if you have any questions or want to meet some new faces. :)
```
Comment (evykassirer) [inline content/F2016/F16-Office-Hours.md]: 80 characters max per line please :)

probably don't need to capitalize Office on this line

love the blurb

File: content/F2016/F16-Office-Hours.md [hunk 4]
```diff
@@ -3,7 +3,10 @@ Date: 2016-10-06 19:00
...
+You can also read more about what our office provides on our [contact us](https://wics-site-pr-267.herokuapp.com/contact-us/) page.
```
Comment (evykassirer) [inline content/F2016/F16-Office-Hours.md]: this link is temporary actually - it should be `{filename}/pages/contact-us/`  I think and that will work with any base url

**Comment by evykassirer** [comment]
https://wics-site-pr-270.herokuapp.com/

does it look the way you were expecting now? I was thinking bullet points like

Monday:
- 10am - 1 pm
- 2pm - 4 pm

```
Monday:

- 10am - 1 pm
- 2pm - 4 pm
```

would be nice

**Comment by fboxwala** [comment]
@rolinawu 
we didn't catch this when Fatima made her PR to delete the calendar, but the link to "this calendar" under office on [this page](http://wics.uwaterloo.ca/contact-us/) is broken. Can you change the sentence to "The schedule for this term's office hours is here" and link the word "here" to your post?

File: content/F2016/F16-Office-Hours.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+ 2pm - 4 pm
+ 
+Thursday:
+ 2pm - 4 pm
+
+Friday:
+10am - 11am
+ 2pm - 4 pm
+
+No office hours will be hold on Saturday and Sunday.
```
Comment (shivangipatwardhan2018) [inline content/F2016/F16-Office-Hours.md]: Hey quick note:
I think it should be "No office hours will be held on Saturday and Sunday"
Thanks!

**Comment by shivangipatwardhan2018** [comment]
Hey quick note: I think it  should be:  "No office hours will be **held** on Saturday and Sunday." :)

**Comment by rolinawu** [comment]
All suggestions updated. Do we still need a blurb for the page?

File: content/F2016/F16-Office-Hours.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+Title: Office Hours for Fall 2016
+Date: 2016-10-06 19:00
```
Comment (evykassirer) [inline content/F2016/F16-Office-Hours.md]: update this date so it shows up above the more recent event on our site :)

**Comment by evykassirer** [comment]
looks good! just one more comment

**Comment by annalorimer** [comment]
@evykassirer Did you have another comment? Or can I get ++'s for merge.

**Comment by evykassirer** [comment]
I had a comment from a week ago that hasn't been fixed yet

**Comment by evykassirer** [comment]
👍 lgtm

**Comment by fboxwala** [comment]
++

**Pull request merged**